exploratory_data_analysis/config.py

```diff
@@ -0,0 +1,9 @@
+global_temp = 'data/global_temperature.txt'
```

Comment:
did you change the config? Has it only 9 lines now?

Reply:
no this is my own config i have created, please check fo the folders before assuming that i changes the main stuff

Reply:
Ok. Please just push to main the files that are going to be used in the entire project. If this config is only for your eda leave it out.